src/Console/OutdatedCommand.php

```diff
@@ -0,0 +1,85 @@
+<?php
```

Comment:
Do we really need the `Console` namespace? Maybe we could leave them at root. Since there only is 4 files we don't need extra namespace, at least in my opinion.